awx/main/tasks/receptor.py

```diff
@@ -437,9 +437,9 @@ def _run_internal(self, receptor_ctl):
                         lines = resultfile.readlines()
                         receptor_output = b"".join(lines).decode()
                     if receptor_output:
-                        self.task.runner_callback.delay_update(result_traceback=receptor_output)
+                        self.task.runner_callback.delay_update(result_traceback=f'Worker output:\n{receptor_output}')
```

Comment:
My plan here, now, is that ansible-runner will write the line to `job_explanation`. By doing that, we _might_ echo the output twice in two places. Because of that, we need some better labeling of what data we're showing, so I added this stuff.